README.md

```diff
@@ -48,12 +48,19 @@
         </a>
     </td>
 </tr>
-<!-- add Publications here -->
+<tr>
```

Comment:
I've added the test coverage badge to this branch so marking PR 54 ready for review.